README.md

```diff
@@ -57,21 +58,34 @@ and deploy it.
 # Getting started
 
 Once you have cloned this repository, can get it running by typing: `./mvnw spring-boot:run`
-from the project root directory. You **must** provide three environment variables for database
+from the project root directory. You **must** provide ten environment variables for database
 access configuration:
 
-- `DATABASE_HOST`
-- `DATABASE_PORT`
-- `SERVICE_NAME` (the database's name)
-- `DATABASE_USER`
-- `DATABASE_PASSWORD`
+- five related to the central database (*THE*):
+  - `DATABASE_HOST`
+  - `DATABASE_PORT`
+  - `SERVICE_NAME` (the database's name)
+  - `DATABASE_USER`
+  - `DATABASE_PASSWORD`
+- five related to the local database:
+  - `LOCAL_DATABASE_HOST`
+  - `LOCAL_DATABASE_PORT`
+  - `LOCAL_SERVICE_NAME` (the database's name)
+  - `LOCAL_DATABASE_USER`
+  - `LOCAL_DATABASE_PASSWORD`
 
 Then head to http://localhost:8090/actuator/health to check if the system was successfully launched:
 the `status` property should have the value *UP*.
 
 Before writing your first line of code, and learn more about the checks, including
 tests, please take a moment and check out our [CONTRIBUTING](CONTRIBUTING.md) guide.
 
+## Database versioning
+
+Our local database is versioned using Flyway. It's a good idea to execute a migration every time you
+run a new version of the application by executing `./mvnw flyway:migrate` providing environment
+variables related to the local database.
+
```

Comment:
It seems there's a filename convention. It should be added here as well. I tried to run, but instead I had this message: `2022-11-29 20:08:42.035  INFO 12282 --- [           main] o.f.c.i.resource.ResourceNameValidator   : 1 SQL migrations were detected but not run because they did not follow the filename convention.`

Reply:
Agreed. Truth is, the whole section has to be modified since I wrote it when I planned to use the Maven plugin.